ocp_resources/resource.py

```diff
@@ -76,6 +83,29 @@ def _get_api_version(dyn_client: DynamicClient, api_group: str, kind: str) -> st
     return res.group_version
 
 
+@contextlib.contextmanager
+def get_retry_cluster_exceptions_client(
+    client: DynamicClient,
+    backoff_factor: float = 0.5,
+    retry_count: int = 5,
+    status_codes=None,
```

Comment:
```suggestion
    status_codes: set | None = None,
```

please add docstring